src/main/java/org/zalando/nakadi/controller/EventTypeController.java

```diff
@@ -186,4 +190,8 @@ private void setDefaultEventTypeOptions(final EventTypeBase eventType) {
         return Responses.create(Response.Status.SERVICE_UNAVAILABLE, exception.getMessage(), request);
     }
 
+    private boolean isNotAdmin(final Client client) {
```

Comment:
I'm a bit skeptical about naming boolean methods `isNotSomething`. Because when you will need to check for `something` you will have 'if (!isNotSomething()) {...}'. I think the more clear approach is to have `isSomething` method, even if you use it in only place with inverse check.